### DISCUSSION TITLE
Logs tab under System Status?

### DISCUSSION THREAD

**Discussion by aleks372** [NightscoutFoundation/xDrip#2128] [Ideas]
Hi,

Sometimes, debuging or connecting watch requires jumping between logs and system status, which one is really time consuming and requires at least 3 steps.

Would be a great if there would be an option to enable logs view by adding extra tab under System Status, that would be so much easier ☺️

![20220516_173536](https://user-images.githubusercontent.com/72258515/168617754-ec72b109-a457-48ad-abb1-de3b8a014380.jpg)

**Comment by Navid200** [comment]
Very interesting.

An alternative would be to have the 3-dot menu button also available on system status screens.

**Comment by Lupum1001** [comment]
Pretty sure the 3-dot menu approach is really easy to do. I think it's just a matter of uncommenting the following line in MegaStatus.java:
https://github.com/NightscoutFoundation/xDrip/blob/e557cba811df35ccdeea38801ab9df5e782f7c5a/app/src/main/java/com/eveningoutpost/dexdrip/MegaStatus.java#L433
and then copying the menu item over into `menu_mega_status.xml`

---

I was initially concerned that the 3-dot solution wouldn't be any faster when you want to _exit_ the log, because I was picturing the current scenario where you have to re-launch System Status from the home screen, but I think the back button would just return you to the Status activity, not the Home screen...

@aleks372 would that be an acceptable solution to you?

**Reply by aleks372** [reply]
If this is already implemented then ofcourse thats would be great to use 🙂

**Comment by Lupum1001** [comment]
Just created a pull request with these changes: https://github.com/NightscoutFoundation/xDrip/pull/2159

The only slight oddity is that the events log itself still has that "return to home screen" button. A more generic 'back' button is probably warranted at some point. As it stands, you can still use the stock back button in android, and it'll take you back to the previous activity, whichever that was.
